airflow/models/taskmixin.py

```diff
@@ -114,6 +115,14 @@ class DAGNode(DependencyMixin, metaclass=ABCMeta):
     def node_id(self) -> str:
         raise NotImplementedError()
 
+    @property
+    def label(self) -> Optional[str]:
+        tg: Optional["TaskGroup"] = getattr(self, 'task_group', None)
+        if tg and tg.node_id and tg.prefix_group_id:
+            # "task_group_id.task_id" -> "task_id"
+            return self.node_id[len(tg.node_id) + 1 :]
```

Comment:
```suggestion
            return self.node_id.split(".", 1)[-1]
```

Reply:
This doesn't feel safe -- there are cases where there can be a dot in the task_id. There _shouldn't_ be, but I cant guarantee it.

Reply:
Makes sense